content/blog/en/version-6-dojo.md

```diff
@@ -0,0 +1,295 @@
+
+![experimental speed demo](assets/blog/version-6-dojo/speed.gif)
+
+This shows an approximate saving of over 2 seconds building the [Dojo RealWorld example](https://github.com/dojo/examples/tree/master/realworld) with the experimental speed mode enabled. On larger Dojo projects we've witnessed more significant savings with development build times dropping by more than 50% having previously taking over 40 seconds, reducing to under 20 seconds using the new speed mode.
+
+## Glob Support For Code Splitting
+
+The `.dojorc` configuration for `bundles` has been enhanced to support globs. This is especially useful for scenarios such as internationalization, meaning that you don't have to define all the language bundles explicitly, instead simply define a matching pattern for each of the locales.
+
+Consider a project with a folder structure that defines language bundles in locale named directories, ideally build tool would create a single bundle for each locale that would be loaded when users change locale. Using a glob provides a low maintenance and minimal effort way to do this over explicitly defining each language bundle modules in the `.dojorc`.
+
+Current configuration required to create a bundle per locale:
+
+> .dojorc
+```json
+{
+  "build-app": {
+    "bundles": {
+      "fr": [ 
+        "src/widgets/home/nls/fr/home",
+        "src/widgets/menu/nls/fr/menu",
+        "src/widgets/blog/nls/fr/blog",
+        "src/widgets/reference/nls/fr/reference"
+      ],
+      "jp": [ 
+        "src/widgets/home/nls/jp/home",
+        "src/widgets/menu/nls/jp/menu",
+        "src/widgets/blog/nls/jp/blog",
+        "src/widgets/reference/nls/jp/reference"
+      ]
+    }
+  }
+}
+```
+
+Using the glob configuration to create a bundle per locale:
+
+> .dojorc
+```json
+{
+  "build-app": {
+    "bundles": {
+      "fr": [ "**/fr/*" ],
+      "jp": [ "**/jp/*" ]
+    }
+  }
+}
+```
+
+## Update to TypeScript
+
+The minimum required TypeScript version has been updated to 3.4. Updating the core framework to use a recent version enables us to leverage the latest features that underpin the function-based widget and middleware typings.
+
+## Revamped Doc Website
+
+// TODO talk about the creation of the new doc website - using BTR and Blocks
+
+## Migration
+
+As usual all of the breaking changes introduced in Dojo 6 are carefully considered, so that we truly believe the benefits outweigh the upgrade effort. To assist with the transition we have updated the CLI upgrade command, which will automatically upgrade your Dojo dependencies, upgrade your application code where possible and highlight areas in the application that require manual intervention. For more information on what has changed in Dojo 6, please see the [migration guide](https://github.com/dojo/framework/blob/master/docs/V6-Migration-Guide.md).
```

Comment:
```suggestion
As usual all of the breaking changes introduced in Dojo 6 are carefully considered, so that we truly believe the benefits outweigh the upgrade effort. To assist with the transition we have updated the CLI upgrade command, which will automatically upgrade your Dojo dependencies, upgrade your application code where possible and highlight areas in the application that require manual intervention. For more information on what has changed in Dojo 6, please see the [Dojo version 6 migration guide](https://github.com/dojo/framework/blob/master/docs/V6-Migration-Guide.md).
```

Reply:
@dylans Do we need to repeat dojo 6 three words after saying dojo 6?

Reply:
My main reason for making sure we include Dojo or Dojo version 6 in the links is to help search engines with the relevancy of the links, and to help with link scannability. We could of course remove it from earlier instances, though it probably does help with anyone trying to translate this content also.